docker/linux/clang-3-9/Dockerfile

```diff
@@ -24,7 +24,12 @@ RUN apt update -qqy \
     && add-apt-repository -y ppa:savoury1/backports \
     && apt update -qqy \
     && apt install -qqy --no-install-recommends git nasm \
+        python3.8 python3.8-dev python3.8-venv \
     && /opt/clean-after-apt.sh
 
+RUN python3.8 -m venv /python38_env
```

Comment:
I think here and in the other Dockerfile there should be a TODO with a bug to follow up on upgrading the images to a later version with python 3.8+ as the default instead of doing things this way